frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -14,6 +14,7 @@ import { GitImportFormData } from './import-types';
 export const createImageStream = (
   formData: GitImportFormData,
   { metadata: { name: imageStreamName } }: K8sResourceKind,
+  dryRunOpt: { queryParams?: { [key: string]: string } } = {},
```

Comment:
Simplify this. The parameter as i can see never changes, so we can have a const variable for this parameter that all create functions use based on the boolean value of `dryRun`.

Reply:
Done